integration/templates/chart_operator_values.go

```diff
@@ -6,7 +6,7 @@ package templates
 const ChartOperatorValues = `cnr:
   address: http://cnr-server:5000
 clusterDNSIP: 10.96.0.10
-e2e: true
+e2e: false
```

Comment:
This is so the priority class is not created. Since EnsureTillerInstalled does this.